internal/measurexlite/resolver.go

```diff
@@ -0,0 +1,106 @@
+package measurexlite
+
+//
+// Trusted Recursive Resolver
+//
+
+import (
+	"context"
+	"net"
+
+	"github.com/ooni/probe-cli/v3/internal/model"
+	"github.com/ooni/probe-cli/v3/internal/netxlite"
+)
+
+// NewTrustedRecursiveResolver2 returns a new trusted recursive resolver
```

Comment:
```suggestion
// NewTrustedRecursiveResolver2 returns a new trusted recursive resolver.
```

Reply:
You should document that `logger` is `MANDATORY` while `URL` is `OPTIONAL` and you should say what happens if you provide an empty `URL`. You should probably say, for future robustness, that we're going to choose a suitable URL (or you could also be specific, up to you).